app/graphql/types/user_type.rb

```diff
@@ -0,0 +1,11 @@
+module Types
+  class UserType < Types::BaseObject
+    field :id, ID, null: false
+    field :first_name, String
+    field :last_name, String
+    field :email, String
+    field :created_at, GraphQL::Types::ISO8601DateTime, null: false
+    field :updated_at, GraphQL::Types::ISO8601DateTime, null: false
+    field :user_role, UserRoleEnum, null: false
```

Comment:
Why `user_role`? It doesn't work

Reply:
i think that class UserRoleEnum work only with this variable